bridges/otelzap/core.go

```diff
@@ -167,7 +183,11 @@ func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
 		r.AddAttributes(attrbuf...)
 	}
 
-	o.logger.Emit(o.ctx, r)
+	logger := o.logger
+	if ent.LoggerName != "" {
+		logger = o.provider.Logger(ent.LoggerName, o.opts...)
```

Comment:
Nit, but how about extracing this logic to get a logger into a private method, to avoid repetition?

Reply:
Thought about it as well, but I decided to follow the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) 😉